src/app/project/commenting-tab/commenting-tab.component.html

```diff
@@ -13,15 +13,15 @@
 
         <div *ngIf="commentPeriods.length > 0">
             <div class="d-flex justify-content-between list-group-item"
-                 *ngFor="let commentPeriod of commentPeriods"
-                 (click)=goToCP(commentPeriod)>
-                <div *ngIf="commentPeriod.commentPeriodStatus">
+                 *ngFor="let commentPeriod of commentPeriods">
+                <div *ngIf="commentPeriod.commentPeriodStatus" class="engagement-tab-item-description">
                     <label class="title">{{commentPeriod.commentPeriodStatus}}</label>
                     <div class="commentInfo">{{commentPeriod.instructions}}</div>
                     <p class="mb-0">{{commentPeriod.dateStarted | date:'MMMM d'}} - {{commentPeriod.dateCompleted | date:'longDate'}}</p>
                 </div>
-                <div>
-                    <button class="badge badge-secondary p-3" aria-label="View details and submit comments to this project's comment period.">View details</button>
+                <div class="engagement-tab-buttons">
+                    <button (click)=goToCP(commentPeriod) class="badge badge-secondary p-3" aria-label="View details of this project's comment period.">View details</button>
+                    <button (click)=addComment() *ngIf="commentPeriod.commentPeriodStatus === 'Open'" class="badge badge-secondary p-3" aria-label="Participate now in this project's comment period.">Participate now</button>
```

Comment:
Looks like a good change, but if there is more than 1 comment period it may be strange to refer to them both as "this project's comment period", which implies exclusivity. Consider the following alternative:
```suggestion
                    <button (click)=goToCP(commentPeriod) class="badge badge-secondary p-3" aria-label="View details of this comment period.">View details</button>
                    <button (click)=addComment() *ngIf="commentPeriod.commentPeriodStatus === 'Open'" class="badge badge-secondary p-3" aria-label="Participate now in this comment period.">Participate now</button>
```
Also, if comment periods have a `name` or `description` field or similar distinguishing characteristic, it would be helpful to render that here

Reply:
Suggestions implemented, thank you